rtl.css

```diff
@@ -1793,6 +1793,86 @@ blockquote, q {
   .search article .entry-meta .author a {
     padding-right: 0px; }
 
+
+ul {
```

Comment:
@PremalathaKumar This is very generic selector. It means it will affect all the `ul` in the theme. Is this intended behaviour?